src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -13,6 +13,8 @@
      * <p>Это задание необходимо решать с использованием логических операторов.</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+
+        return ((a | b)^(a | c)^(a | d)^(b | c)^(b | d)^(c | d)) & ((a & b)^(a & c)^(a & d)^(b & c)^(b & d)^(c & d));
```

Comment:
some of checks were failed. Lets fix it
![image](https://user-images.githubusercontent.com/36365171/70029667-438d9100-15b0-11ea-92fa-85b23eb0f315.png)